cpp/tests/utilities/cudf_test_fixtures.h

```diff
@@ -48,3 +50,34 @@ struct GdfTest : public ::testing::Test
         ASSERT_RMM_SUCCEEDED( rmmFinalize() );
     }
 };
+
+struct TempDirTestEnvironment : public ::testing::Environment
```

Comment:
Please document what this is. (comment).

Reply:
added doxygen style documentation